include/ziapi/Version.hpp

```diff
@@ -8,9 +8,34 @@ namespace ziapi {
 struct Version {
     Version(int major, int minor) : major(major), minor(minor) {}
 
-    bool operator==(const Version &other) const { return this->major == other.major && this->minor == other.minor; }
+    inline bool operator==(const Version &other) const noexcept
+    {
+        return this->major == other.major && this->minor == other.minor;
```

Comment:
```suggestion
        return major == other.major && minor == other.minor;
```
I'd remove the `this` since it is not necessary in C++ and I find it more readable without it.
Anyway it is not a big deal just preferences IMO.